firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -80,7 +80,31 @@ const TCMD_TelecommandDefinition_t TCMD_telecommand_definitions[] = {
         .number_of_args = 0,
         .readiness_level = TCMD_READINESS_LEVEL_FOR_OPERATION,
     },
-
+    {
+        .tcmd_name = "set_int_config_var",
```

Comment:
Recommendation: I know it feels a little weird, but it might be best to rename these all to be in the format of `config_`, like `config_set_int_var`, `config_get_str_var`, etc.